esbuild.server.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Aside: Why is `typescript-eslint` interfering in `.js` files?

Reply:
Good catch! I've done some digging and I think we'll need to make some tweaks to our eslintrc to only use `typescript-eslint` for ts/tsx files. I'm thinking it'd make a good followup PR? https://github.com/typescript-eslint/typescript-eslint/issues/109#issuecomment-536160947